### DISCUSSION TITLE
Automatically match width/height of cards

### DISCUSSION THREAD

**Discussion by olssonks** [zauberzeug/nicegui#997] [Q&A] [answered]
### Question

What's the proper way to get cards match dimensions automatically? I've tried using `classes('w-full h-full')`, which usually works for width but not for height. For example the two cards in the bottom row are slightly different heights (red arrow).

```py
@ui.page('/Pulse_Streamer')
def main():
    with ui.card():
        with ui.column():
            with ui.card().classes('w-full h-full'):
                fig = go.Figure()
                ps_gui_holder.seq_plot = ui.plotly(fig).classes('w-full h-full')
            with ui.row().classes('w-full h-full no-wrap'):
                with ui.card():
                    edit_channel_card()
                with ui.card():
                    edit_pulse_card()   
    return
```

![alignment_image](https://github.com/zauberzeug/nicegui/assets/62556548/9ff91670-0c31-4da5-a0b9-09d7693f8134)

**Comment by falkoschindler** [comment, marked as answer]
Hi @olssonks,
You can use the Tailwind class "items-stretch" on the row. This will let all child elements fill the vertical space spanned by the row: https://tailwindcss.com/docs/align-items

**Reply by olssonks** [reply]
Thank you!
